core/tests/protractor_utils/collectionEditor.js

```diff
@@ -27,6 +27,20 @@ var addExistingExploration = function(explorationId) {
   element(by.css('.protractor-test-add-exploration-button')).click();
 };
 
+// Search and add existing exploration to the node graph.
+var searchExistingExploration = function(query) {
```

Comment:
I think this should be searchForAndAddExistingExploration. Just "searching" alone does not convey selection.